scripts/gen_jvt.py

```diff
@@ -95,7 +90,7 @@ def __init__(
 
     def generate(self, download, jobs):
         """Generates the test suite and saves it to a file"""
-        new_test_vectors = []
+        multiple_test_vectors = []
```

Comment:
This one should stay as it was to avoid variable shadowing inside function `_fill_checksum_h264_multiple()`
```suggestion
        new_test_vectors = []
```